samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _fmfu_smp_svr_sample:
+
+Full modem update over SMP Server Sample
```

Comment:
"over SMP Server" ? Should it be "over SMP" ? or "using SMP" ? I'm not sure about the SMP terminology, so I might be very wrong :)

Reply:
So MCUMgr uses SMP sort of as a server which is what is being used in examples in zephyr. Where they svr is the short form of server.

Reply:
Anyways changed it to full modem firmware update using SMP